Documentation/ApiOverview/Typo3CoreEngine/Database/Index.rst

```diff
@@ -172,19 +172,19 @@ Command keywords and values
          Value is an uid of the :php:`sys_language` to localize the record into.
          Basically a localization of a record is making a copy of the record
          (possibly excluding certain fields defined with :php:`l10n_mode`) but
-         changing relevant fields to point to the right sys language / original
+         changing relevant fields to point to the right :php:`sys_language` / original
```

Comment:
This must be `:sql:` instead.

Reply:
It depends on whether it's a TCA option like `l10n_mode` or simply an SQL field name like `sys_language`. There is no TCA option for the latter.

Reply:
In case you are confused, just comment it and I'll push the right version, that are easy made changes for me and I hopefully am able to distinguish these ;)

`:sql:` is not available yet, but could be added to the `Includes.txt`.

Reply:
afaik, :sql: is not available, so we stick to :php: here for now.